### DISCUSSION TITLE
Quick Search option in Pimcore 11

### DISCUSSION THREAD

**Discussion by Amarjeet664** [pimcore/pimcore#15984] [Q&A / Support] [answered]
Hi Team,

Looks like Quick search functionality is not available out of the box in Pimcore 11. 
Need to install through some package, Can anyone let me know the package name.

I am talking about the below mentioned button

![image](https://github.com/pimcore/pimcore/assets/21308395/61f80d06-df6b-4023-a571-31fa27394e0c)

**Comment by Corepex** [comment, marked as answer]
Hi @Amarjeet664, the bundle your looking for is the `SimpleBackendSearchBundle`. The bundle isn't extracted yet so it ships with the core bundle. https://github.com/pimcore/pimcore/tree/11.x/bundles/SimpleBackendSearchBundle

If you don't see the option please check if the bundle is enabled in your `bundles.php` and installed :)

**Reply by Amarjeet664** [reply]
Thanks @Corepex for the reply
I found the reason for it to be not working. I checked `SimpleBackendSearchBundle` was already present in my source code, the only issue was I didn't enabled it.

Worked fine after running the bundle enable command
`bin/console pimcore:bundle:install PimcoreSimpleBackendSearchBundle
`

**Reply by Corepex** [reply]
nice - so it was just not installed :+1:

**Comment by doncatnip** [comment]
Unfortunately, the solution given here does not work for me in Pimcore 11.1.4

This is my config/bundles.php:
```
<?php

use Pimcore\Bundle\DataHubBundle\PimcoreDataHubBundle;
use Pimcore\Bundle\SimpleBackendSearchBundle\PimcoreSimpleBackendSearchBundle;


return [
    PimcoreDataHubBundle::class => ['all' => true],
    PimcoreSimpleBackendSearchBundle::class => ['all' => true],
];
```

Now even if I run pimcore:bundle:list it claims the following:
`  You have requested a non-existent service "Pimcore\Bundle\SimpleBackendSearchBundle\Installer".`
